api/v1beta2/ibmpowervscluster_types.go

```diff
@@ -45,13 +43,61 @@ type IBMPowerVSClusterSpec struct {
 	// ControlPlaneEndpoint represents the endpoint used to communicate with the control plane.
 	// +optional
 	ControlPlaneEndpoint capiv1beta1.APIEndpoint `json:"controlPlaneEndpoint"`
+
```

Comment:
Now we need to use DHCP's private network here. Please see if you want to document that in the comment.

Reply:
Another design suggestion:
When there is a need for creating dhcp server you might need to check for existing dhcp server and compare the CIDR and chose a different one.
Rather you can use the existing one instead of creating new one which will keep things simpler.

Reply:
Yeah, Makes sense will document it accordingly, may be the flow should be like
```
If the networkID is empty, check for existing DHCP server and use it or else create new one.
```